ddtrace/tracer/tracer.go

```diff
@@ -33,18 +33,15 @@ type tracer struct {
 	*config
 	*payload
 
-	// exitChan requests that the tracer stops.
-	exitChan chan struct{}
-
 	// payloadChan receives traces to be added to the payload.
 	payloadChan chan []*span
 
-	// stopped is a channel that will be closed when the worker has exited.
-	stopped chan struct{}
-
 	// climit limits the number of concurrent outgoing connections
 	climit chan struct{}
 
+	// stop causes the tracer to shut down when closed.
```

Comment:
```
// stop signals the tracer to shutdown.
```
Because it can happen both when sending down the channel and also when closed.

Reply:
Sending on the channel will cause one of the worker goroutines to exit, but not the others. It will trigger a partial shutdown of the tracer.

Reply:
Ah yes, of course.